src/app/Library/CrudPanel/FieldGroup.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel;
+
+class FieldGroup
+{
+    protected $fields;
+
+    public function __construct(...$fields)
+    {
+        $this->fields = $fields;
+    }
+
+    // -------------
+    // MAGIC METHODS
+    // -------------
+
+    /**
+     * If a developer calls a method that doesn't exist, assume they want:
+     * - the CrudField object to have an attribute with that value;
+     * - that field be updated inside the global CrudPanel object;.
+     *
+     * Eg: type('number') will set the "type" attribute to "number"
+     *
+     * @param  string $method     The method being called that doesn't exist.
+     * @param  array $parameters  The arguments when that method was called.
+     *
+     * @return CrudField
+     */
+    public function __call($method, $parameter)
+    {
+        foreach ($this->fields as $field) {
+            $field->setAttributeValue($method, $parameter[0]);
```

Comment:
Mb call method directly?
```
$field->{$method}($parameter[0]);
````

Reply:
Might be better, indeed. I think that would also be helpful with using Helpers (aka Convenience Methods) like `size()` or `link()`. Methods that aren't actual attributes, but that themselves set one/more attributes.

Reply:
Just did it guys. Thanks for pointing that out :)